src/ResourceManager/AzureBatch/Commands.Batch/Locations/GetBatchLocationQuotasCommand.cs

```diff
@@ -18,8 +18,8 @@
 
 namespace Microsoft.Azure.Commands.Batch
 {
-    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchSubscriptionQuotas), OutputType(typeof(PSBatchSubscriptionQuotas))]
-    public class GetBatchSubscriptionQuotasCommand : BatchCmdletBase
+    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchLocationQuotas), OutputType(typeof(PSBatchLocationQuotas))]
```

Comment:
@cormacpayne 
Yes, the management lib had breaking changes, which results in breaking changes here.
I'll add an alias for the old name, but can you elaborate on what you're looking for with respect to the output type?